node_modules/oae-authentication/lib/rest.js

```diff
@@ -44,3 +43,113 @@ require('./strategies/twitter/rest');
+ * @HttpResponse                404                     Config value ... does not exist
+ */
+OAE.globalAdminRouter.on('post', '/api/auth/local/reset/init/:tenantAlias/:username', _getResetPasswordSecret);
+
+/**
+ * @REST LocalResetInit
+ *
+ * generate an user token based on a existing username
+ *
+ * @Server      Tenant
+ * @Method      GET
+ * @Path        /auth/local/reset/init/{username}
+ * @PathParam   {string}        username     			The username for the user
+ * @Return      {string}
+ * @HttpResponse                200                     Configuration value cleared
+ * @HttpResponse                400                     Missing configuration. Example configuration: `"oae-authentication/twitter/enabled"`
+ * @HttpResponse                400                     Missing tenant alias
+ * @HttpResponse                400                     You cannot mix clearing an entire element and an optionalKey
+ * @HttpResponse                401                     Only authorized tenant admins can change config values
+ * @HttpResponse                401                     User is not allowed to update config value ...
+ * @HttpResponse                404                     Config value ... does not exist
+ */
+OAE.tenantRouter.on('get', '/api/auth/local/reset/init/:username', _getResetPasswordSecret);
+
+
+var _resetPassword = function(req, res){
+	AuthenticationAPI.resetPassword(req.ctx, req.params.tenantAlias, req.params.username, req.params.secret, req.body.newPassword, function(err, changed) {
+	    if (err) {
+	        return callback(err);
+	    }
+		
+		return res.send(200);
+	})
+};
+
+/**
+ * @REST GlobalResetChange
+ *
+ * Reset an user's password based on a existing username
+ *
+ * @Server      admin
+ * @Method      POST
+ * @Path        /auth/local/reset/change/{tenatAlias}/{username}
+ * @PathParam   {string}        username                The username for the user
+ * @FormParam   {string}        newPassword             The password for the user
+ * @FormParam   {string}        secret                  The secret for the user
+ * @Return      {void}
+ * @HttpResponse                200                     Configuration value cleared
+ * @HttpResponse                400                     Missing configuration. Example configuration: `"oae-authentication/twitter/enabled"`
+ * @HttpResponse                400                     Missing tenant alias
+ * @HttpResponse                400                     You cannot mix clearing an entire element and an optionalKey
+ * @HttpResponse                401                     Only authorized tenant admins can change config values
+ * @HttpResponse                401                     User is not allowed to update config value ...
+ * @HttpResponse                404                     Config value ... does not exist
+ */
+OAE.globalAdminRouter.on('post', '/api/auth/local/reset/change/:tenantAlias/:username/', _resetPassword);
```

Comment:
This one isn't really necessary as global admins can already change a user's password through another endpoint